### DISCUSSION TITLE
jeo-carto build issue

### DISCUSSION THREAD

**Discussion by CodeK1988** [mapsforge/vtm#1144] [Q&A] [answered]
hi mate,  com.github.jeo.jeo:jeo-carto:master-SNAPSHOT, build is very slow, can it be integrated offline or maven?  i have jeo-carto-0-SNAPSHOT.jar file.

**Comment by devemux86** [comment, marked as answer]
`vtm-jeo` is currently disabled in the build because of this reason.
Aren't there dependencies in a better place?

You can search online about how to use local jars in the Gradle build.
